autoload/fern/renderer/nerdfont.vim

```diff
@@ -177,5 +160,5 @@ call s:Config.config(expand('<sfile>:p'), {
       \ 'padding': ' ',
       \ 'root_symbol': '',
       \ 'indent_markers': 0,
-      \ 'root_leading': 1,
+      \ 'root_leading': ' ',
```

Comment:
One more.

`root_leading` should refer `leading` in default so it should be defined without `s:Config` like

```vim
call s:Config.config(expand('<sfile>:p'), {
      \ ...
      \})

let g:fern#renderer#nerdfont#root_leading = get(g:, 'fern#renderer#nerdfont#root_leading', g:fern#renderer#nerdfont#leading)
```